### ISSUE TITLE
Annual dotnet/runtime community survey 2023

### ISSUE THREAD

**Issue by danmoseley** [dotnet/runtime#81866]
Hello everyone, we'd like to hear from you in this year's repo survey. Whether you open or comment on issues, or contribute PR's, please help this repo get better.

Here's the survey link -- you can complete it for up to 3 repos: 
[https://www.surveymonkey.com/r/8KTX9VT?Source=RepoLink](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.surveymonkey.com%2Fr%2F8KTX9VT%3FSource%3DRepoLink&data=05%7C01%7CDan.Moseley%40microsoft.com%7Ca7e830e18f1e4b61041508db0a373e7d%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638115004691949601%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NrT%2B0A8YGEu21XM7mwLPyFPp4pbeeMwIi0Ksarll080%3D&reserved=0)

We've run this several years and it helps us find areas to improve and compare best practices across repos. Here are the results we shared from last year's survey: https://github.com/dotnet/runtime/issues/67766

**Comment by danmoseley** [comment]
cc @JulieLeeMSFT

**Comment by En3Tho** [comment]
Please add JetBrains Rider as a legit option for the IDE. To me it seems like intentional ignore due to some kind of pressure from MS or VS team but I belive runtime repo maintainers should strive to be unbiased as much as they can.

**Comment by Gnbrkm41** [comment]
@En3Tho, I do believe adding support for Rider would be great, but at the same time I'm not sure how easy of a task that would be because of all the weird and obscure things the runtime repo uses to allow something like building everything from scratch - I'd imagine there's a fair bit of customised parts even built into the VS, so it might also require cooperation from the Rider team (which I'm not sure if they'd be willing to do).

Also, I found that the runtime repo is very quick at adapting new (sometimes experimental) language features in the library code before any sort of proper public beta / release - My past experience with Rider (& R#) is that they're quite slow at adding supports for those, so even if all those things happen it might still be very hard to use Rider on the runtime repo.

Just my thoughts.

**Comment by En3Tho** [comment]
@Gnbrkm41 No, didn't mean fully supporting Rider as IDE for working in dontet/runtime. First of all it already works: I did jit contributions using Rider on Windows. Obviously new C# bits are VS preview only because Rider is not using Roslyn but their own language service.

What I want is Rider being recognized as a proper IDE choice in surveys like these.

**Comment by Gnbrkm41** [comment]
@En3tho - Oops, my bad. I agree on that part.

**Comment by michaldobrodenka** [comment]
> NATIVEAOT+ARM32!!!

I support this idea as it would reduce ram requirements for small programs. ARM32 is widely used in embedded segment with linux devices.
